docs/sphinx/source/whatsnew/v0.8.0.rst

```diff
@@ -11,6 +11,7 @@ Enhancements
 
 Bug fixes
 ~~~~~~~~~
+* Fixed unit and default value errors in :py:func:`pvlib.soiling.hsu`. (:pull:`XXX`)
```

Comment:
Missed the PR number in the whatsnew entry

Reply:
I pulled the trigger early

Reply:
I just submitted PR980 that addresses @kanderso-nrel 's comment about arbitrary time intervals.  Next I will submit a demonstration of the hsu function showing that it matches one of the figures in the original JPV paper.